src/main/java/com/ubc/cpsc304/hotelify/entity/Hotel.java

```diff
@@ -0,0 +1,40 @@
+package com.ubc.cpsc304.hotelify.entity;
+
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+/**
+ * The entity class for hotel.
+ * Created by Scott on 2018-11-5
+ */
+
+@Data
+@Entity
+public class Hotel {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(nullable = false, updatable = false, unique = true)
+    private Long id;
+
+    @Column(nullable = false, columnDefinition = "VARCHAR(30) DEFAULT ''")
+    private String brandName;
+
+    @Column(columnDefinition = "VARCHAR(30) DEFAULT ''")
+    private String branchName;
+
+    @Column
+    private int propertyClass;
+
+    @Column(nullable = false)
```

Comment:
you would need the `@ManyToOne` annotation here

Reply:
The `Address` entity is in the master now, so just Pull the master and use `@ManyToOne private Address address`. IT SHOULD WORK. If you check the db, the foreign key should be called: "address_id" by default, so you do not need to explict write it (or if you want to in the `@ManyToOne` annoatation, use `mappedby`)